package.json

```
@@ -22,7 +22,9 @@
     "js-yaml": "^3.13.1",
     "listr": "^0.14.3",
     "rimraf": "^3.0.0",
-    "tslib": "^1.10.0"
+    "tslib": "^1.10.0",
+    "yargs": "14.0.0",
+    "@sap/cloud-sdk-generator": "1.14.0"
```

This should not be a fixed dependency so we make sure it keeps working with newer SDK versions.
```suggestion
    "@sap/cloud-sdk-generator": ">=1.14.0"
```